todo-frontend-angular/src/app/todolist-selector/todo/todo.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Todo } from 'src/app/models/todo.model';
+
+@Component({
+  selector: 'app-todo',
+  templateUrl: './todo.component.html',
+  styleUrls: ['./todo.component.scss']
+})
+export class TodoComponent implements OnInit {
+  deadline: string = '';
+  constructor() { }
+
+  @Input() todo?: Todo;
+
+  @Output() onCheckboxClick = new EventEmitter<boolean>();
+
+  ngOnInit(): void {
+    this.deadline = this.todo?.deadline?.split('T')[0] ?? "";
```

Comment:
Is this a date? :D

Reply:
yes.. replaced with pipe in later pull request